tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/firebase/FirebasePushClient.java

```diff
@@ -115,7 +115,6 @@ private Aps buildApsDictionary(FirebasePushRequest pushRequest) {
                     .setTitle("Twake Mail")
```

Comment:
```suggestion
                    .setTitle("Twake Mail for type EmailDelivery")
```

Do you mean adding something like this, and just to ease the testing on staging?

Reply:
I meant title for this PR :)

Reply:
as it only affects notification payloads that contain EmailDelivery

Reply:
IMO, the default title should only be `Twake Mail` to avoid notification cases from being modified as desired by the client.